src/vehicles/use-vehicles-enabled.ts

```diff
@@ -0,0 +1,18 @@
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {useDebugOverride} from '@atb/debug';
+import {StorageModelKeysEnum} from '@atb/storage';
+
+export const useIsVehiclesEnabled = () => {
+  const [debugOverride] = useVehiclesInMapDebugOverride();
+  const {enable_vehicles_in_map: enabledInRemoteConfig} = useRemoteConfig();
+
+  return debugOverride ?? enabledInRemoteConfig;
```

Comment:
I think you need
```
  if (debugOverride !== undefined) {
    return debugOverride;
  }
  return enabledInRemoteConfig;
```

When `debugOverride` is set to false we want the vehicles functionality to be turned off, and not fallback to the value in RemoteConfig.